### ISSUE TITLE
Uncaught Error: Pane has been destroyed

### ISSUE THREAD

**Issue by isheed** [atom/atom#11186]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.5.4
**System**: Mac OS X 10.11.2
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Pane has been destroyed

```
At /Users/alimaster/Documents/Atom.app/Contents/Resources/app.asar/src/pane.js:631

Error: Pane has been destroyed
    at Pane.module.exports.Pane.activate (/Users/alimaster/Documents/Atom.app/Contents/Resources/app.asar/src/pane.js:631:15)
    at Pane.module.exports.Pane.focus (/Users/alimaster/Documents/Atom.app/Contents/Resources/app.asar/src/pane.js:207:21)
    at atom-pane.<anonymous> (/Users/alimaster/Documents/Atom.app/Contents/Resources/app.asar/src/pane-element.js:50:23)
```

### Commands

```
```

### Config

```json
{
  "core": {},
  "editor": {
    "fontSize": 13,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
color-picker, v2.1.1
emmet, v2.4.2
emmet-simplified, v2.5.0
language-babel, v2.15.8
react, v0.14.1
regex-railroad-diagram, v0.11.0

# Dev
No dev packages
```

**Comment by 50Wliu** [comment]
Hi @isheed, what were you doing when this occurred?

**Comment by eddiemonge** [comment]
I got this right now. I had a few split panes (both horizontal and vertical). I closed a couple of them and one was left hanging.

**Comment by 50Wliu** [comment]
@eddiemonge I suspect you're hitting #10903 (hard to tell when all the tracebacks are the same :frowning:).

Closing this issue as no extra information has been provided.

**Comment by tburkett88** [comment]
I get this issue all the time. When closing the last tab in a pane, sometimes it just leaves a blank pane that is unusable. If i click in that pain or try to drag a new file to that pane, I get that error message.

**Comment by NKjoep** [comment]
I confirm it just happened to me with version `1.7.4`

**Comment by au9ustine** [comment]
Same on `1.7.4`

**Comment by HLFH** [comment]
Same on 1.7.4 bis.

**Comment by jordanshipt** [comment]
same. found a fix i think. will post update later

**Comment by ricklecoat** [comment]
Same here. A precursor seems to be that activating a 'split' (in any direction) does not actually create a visible, usable, new pane -- ie. no new pane containing the requested file appears, although the app appears to think that it has created the pane (and perhaps, on some programming level, it has).This 'ghost' pane causes errors when one attempts to close existing panes down.

**Comment by 50Wliu** [comment]
This issue (or at least the multi-pane exception) should be fixed starting in Atom 1.9.0.  If you got this exception without having multiple panes open, please create a new issue.

**Comment by skamansam** [comment]
I just received the same error in 1.9.0-beta0. This is the first time it has happened to me, and I have been using atom since before 1.0.

**Comment by 50Wliu** [comment]
> I just received the same error in 1.9.0-beta0. This is the first time it has happened to me, and I have been using atom since before 1.0.

Please read my comment right above yours and file a new issue if necessary.

**Comment by BlaM** [comment]
How do I get rid of the blank space I get after the error occured? (1.8.0)

**Comment by 50Wliu** [comment]
@BlaM I don't believe so, sorry.  Reloading the window might work (ctrl-cmd-alt-L on macOS, Ctrl+Alt+R otherwise).

**Comment by oulrich1** [comment]
Hit this in 1.9.2. 

Had like 6 horizontally splits. The right most split was vertically split. I moved the top split of this right most section to the 2nd from the right horizontal section. 

That is when the error occurred.  It left  the top right portion of the window empty. Tried closing the bottom right which still had my file. The entire right section then did not collapse, instead left behind an empty section. 

Closing and opening atom reset the splits.
